content/cloud/changelog/spin-cloud-v0-7-0.md

```diff
@@ -0,0 +1,40 @@
+title = "Spin Cloud v0.7.0 Updates"
+template = "changelog_item"
+date = "2024-01-25T12:00:00Z"
+enable_shortcodes = true
+tags = ["spin_cloud_plugin"]
+[extra]
+type= "changelog_post"
+twitter_card_type = "summary_large_image" 
+image = "/static/image/changelog/spin-cloud-0.7.0.jpg"
+
+---
+
+<img src="/static/image/changelog/spin-cloud-0.7.0.jpg" alt="Spin Cloud 0.7.0 changelog">
+
+Before the v0.7.0 release of the [`spin cloud` plugin](https://github.com/fermyon/cloud-plugin), Spin applications could only use a "default" key-value store. This release of the cloud plugin brings support for sharing key-value stores between applications, linking multiple key-value stores to an application, and dynamically updating which key-value stores are linked to an application.
+
+<!-- break -->
+
+## Spin Cloud v0.7.0 Updates
+
+The updated CLI includes all the commands needed to take these management actions:
+
+`spin cloud kv create` will create a key-value store on Fermyon Cloud
```

Comment:
@tpmccallum just a heads up that this renders as a paragraph instead of a list

Reply:
Ah, nice catch @kate-goldenring 
This will be updated/corrected in the next commit/push.